notNeededPackages.json

```diff
@@ -629,6 +629,12 @@
             "typingsPackageName": "zone.js",
             "sourceRepoURL": "https://github.com/angular/zone.js",
             "asOfVersion": "0.5.12"
+        },
+        {
```

Comment:
`notNeededPackages.json` isn't meant for this purpose. Ping me when this is merged and I'll deprecate `@types/state-machine`.